src/main/java/springcourse/phonebook/model/UserContact.java

```diff
@@ -0,0 +1,38 @@
+package springcourse.phonebook.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "contact")
+public class UserContact {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String contactName;
+    private String phoneNumber;
```

Comment:
Shouldn't we keep list of phone numbers?
In task description it's specified that: 'PUT/{name}; phoneNumber - add phone to existing name' so I assume one contact can have several phone numbers

Reply:
Fixed.
I implemented it as a List of phone numbers.